ui/next.config.js

```diff
@@ -1,4 +1,18 @@
 /** @type {import('next').NextConfig} */
+const CSPDirectives = [
+    "default-src 'self';",
+    "script-src 'self' 'unsafe-eval' 'unsafe-inline';",
```

Comment:
Why unsafe-eval and unsafe-inline? These should be removed. If you need them locally for hot reload, you can specifically add them for dev mode.

Reply:
I based it off of the example on the NextJS docs which also had those - I've now set it to be 'self' only unless local, in which case it also has 'unsafe-eval', which then works OK. Will see how it works on the deployed environment